src/app/components/node/node.component.html

```diff
@@ -1,101 +1,80 @@
+            </div>
+            <div class="label">
+                <span class="full">C</span>G<span class="full">C Genes</span>:
+            </div>
+            <div class="value">
+                <div class="group group_blue">{{ consolidatedVariantInfo.genesCgc.size }}</div>
+                <div *ngIf="!displayNode?.cluster"class="full">/{{ consolidatedVariantInfo.genesAll.size }}</div>
+            </div>
+            <div class="label">
+                D<span class="full">rugs</span>:
+            </div>
+            <div class="value">
+                <div class="group">{{ consolidatedVariantInfo.drugs.size }}</div>
+            </div>
         </div>
 
-        <div class="summary" *ngIf="jsonData?.summary">
+        <div class="summary" *ngIf="!showTable">
             <div class="summary_header">Mutation Edge Info</div>
             <div class="summary_table">
                 <table>
                     <thead>
                         <tr>
-                            <th *ngFor="let column of summaryColumns" [ngClass]="'column_' + column.field">
+                            <th *ngFor="let column of summaryColumns" [ngClass]="'column_' + column.cssSuffix">
                                 {{column.label}}
                             </th>
                         </tr>
                     </thead>
                     <tbody>
-                        <tr *ngFor="let data of jsonData?.summary" (click)="$event.stopPropagation(); onToggleTable(data.tier)">
-                            <td *ngFor="let column of summaryColumns" [ngClass]="'column_' + column.field">
+                        <tr *ngFor="let row of sortedSummaryTableRows" (click)="$event.stopPropagation(); onToggleTable(row)">
+                            <td *ngFor="let column of summaryColumns" [ngClass]="'column_' + column.cssSuffix">
                                 <div class="group group_red">
-                                    {{$any(data)[column.field]}}
+                                    {{ column.accessor(row.consequence, row.variantInfo) }}
                                 </div>
                             </td>
                         </tr>
                     </tbody>
                 </table>
             </div>
-            <div class="summary_button" (click)="$event.stopPropagation(); onToggleTable(undefined)">{{(!phylogenyShowTable) ? 'Open in' : 'Hide'}} Table</div>
+            <div class="summary_button" (click)="$event.stopPropagation(); onToggleTable(null)">{{(!showTable) ? 'Open in' : 'Hide'}} Table</div>
         </div>
     </div>
 
-    <div class="block_background" *ngIf="children && children.length > 0">
+    <div class="block_background" *ngIf="displayNode.children.length > 0">
         <svg width="100%" height="100%" viewBox="0 0 100 100" preserveAspectRatio="none">
-            <linearGradient x1="0" y1="0" x2="100%" y2="0" id="gradient">
```

Comment:
if, for a given instance of this component, we want its .block_background to use its linearGradient, the id here must be unique on the page